test/api/Qemu/qemu_clone_test.go

```diff
@@ -35,6 +35,27 @@ func Test_Clone_Qemu_VM(t *testing.T) {
 
 }
 
+func Test_Clone_Qemu_VM_To_Different_Storage(t *testing.T) {
```

Comment:
currently none of the API test are really functional but this tells us what situation we should have coverage for.